### PR TITLE
Created ScenarioExecutor service:

### PR DESCRIPTION
**Pull request by Lokankara**
- Implemented ScenarioExecutorService class with execute method
- executeSteps that executes the queue action step by step
- added enum StepExecutionType with necessary StepExecution
- created StepExecutionException if StepExecution is not supported

### CODE REVIEW

File: executor-service/src/main/java/executor/service/service/step/UnsupportedStepExecution.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package executor.service.service.step;
+
+import executor.service.exception.StepExecutionException;
+import executor.service.model.Step;
+import org.openqa.selenium.WebDriver;
+
+import static executor.service.service.executor.Action.UNSUPPORTED_ACTION;
+
+public class UnsupportedStepExecution
```
Comment (MazurenkoNick) [inline executor-service/src/main/java/executor/service/service/step/UnsupportedStepExecution.java]: Do we really need a separate StepExecution class for unsupported actions, whose method just throws an exception?

File: executor-service/src/main/java/executor/service/service/executor/StepExecutionType.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    }
+
+    public static StepExecutionType fromString(
+            final String action) {
+        return Arrays
+                .stream(StepExecutionType.values())
+                .filter(type -> type.name()
+                        .equals(action.toUpperCase()))
+                .findFirst()
+                .orElse(UNSUPPORTED);
```
Comment (MazurenkoNick) [inline executor-service/src/main/java/executor/service/service/executor/StepExecutionType.java]: .orElseThrow(() -> new StepExecutionException(String.format("The action %s is not supported.", action));

Comment (MazurenkoNick) [reply executor-service/src/main/java/executor/service/service/executor/StepExecutionType.java]: fail fast approach.

**Review by MazurenkoNick — APPROVED** [review]

File: executor-service/src/main/java/executor/service/service/step/UnsupportedStepExecution.java (hunk 1 above)
Comment (Lokankara) [reply executor-service/src/main/java/executor/service/service/step/UnsupportedStepExecution.java]: Good point, keep it short and simple

**Pull request merged**